### ISSUE TITLE
[Bug]: Steam.melvoridle.com refused to connect

### ISSUE THREAD

**Issue by RealGlobalGenocide** [MelvorIdle/melvoridle.github.io#2249]
### Describe the bug

Hi,

Attempted to launch game but I get stuck with a "loading circle" in the top bar where the Melvor "M" icon is until the "steam.melvoridle.com refused to connect" page appears and the game stops.

### Reproduction Steps

1. Attempt to open game from steam.
2. The game opens but is unable to connect.

### Expected behaviour

The game to launch and connect correctly.

### Screenshots

![Melvor bug1](https://user-images.githubusercontent.com/98965141/156000244-8011806e-0c9f-42a1-8f87-7d64a9b74978.png)
![melvor bug 2](https://user-images.githubusercontent.com/98965141/156000484-d76ac6c6-3d8e-4eb8-907a-fc555a72275a.png)

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [X] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0.3

### Game Subversion

most recent, unable to tell.

### Are you using any scripts or extensions?

No

**Comment by jackchenry** [comment]
Have exactly the same issue.

**Comment by MountainOfBooks** [comment]
Same connection-issue here.
Please note: No issue this morning.

**Comment by MelvorIdle** [comment]
Thank you for your patience. This has now been resolved!

**Comment by minaolenpalm** [comment]
> Thank you for your patience. This has now been resolved!

I had the same error just moments ago. I opened game from steam and did some stuff on my first save. After clicking sign out to switch to another save, I received the white error:
![image](https://user-images.githubusercontent.com/61579764/223192440-d36b9306-dca2-411c-9954-41e34315abd5.png)

When I closed the client and reopened, all seems to be working again and my game was saved prior to receiving the error.

my mods:
![image](https://user-images.githubusercontent.com/61579764/223192742-3f3b25a1-39f4-4e6d-b016-deea45977e19.png)
game V1.1.1 (?5053)
